### PR TITLE
[GHA] Uplift Linux GPU RT version to 22.43.24595.30

### PR DESCRIPTION
**Pull request by bb-sycl**
Scheduled drivers uplift

### CODE REVIEW

**Comment by bader** [comment]
L0 tests pass, which signals that regressions have been fixed.
On the other hand, testing with OpenCL backend failed. It looks like OpenCL driver installation is broken.

**Comment by pvchupin** [comment]
Yes, it seems this 22.43 release was updated. 
Current OpenCL issue is sort of new and caused by packages disconnection between https://github.com/intel/compute-runtime/releases and https://github.com/intel/intel-graphics-compiler/releases.

One of these needs to be updated.

File: devops/dependencies.json [hunk 1]
```diff
@@ -74,4 +74,4 @@
       "root": "{ARCHIVE_ROOT}/comp/oclfpga/win"
     }
   }
-}
+}
```
Comment (bader) [inline devops/dependencies.json:77]:
```suggestion
}

```

**Review by bader — COMMENTED** [review]
> Yes, it seems this 22.43 release was updated. 
> Current OpenCL issue is sort of new and caused by packages disconnection between https://github.com/intel/compute-runtime/releases and https://github.com/intel/intel-graphics-compiler/releases.
> 
> One of these needs to be updated.

Level Zero runtime released ~one week ago, whereas compute runtime ~two weeks ago. I hope next release of compute-runtime will fix all regressions.

**Pull request closed without merging**